### DISCUSSION TITLE
Unable to run VM

### DISCUSSION THREAD

**Discussion by ABBailey1981** [quickemu-project/quickemu#1386] [Quickemu]
I have been trying for a couple days to install and configure QuickEMU without success.

The initial quickget command works just fine but running the quickemu --vm [vmname].conf produced the following

quickemu --vm macos-sonoma.conf
Error: couldn't find RGB GLX visual or fbconfig
Quickemu 4.9.5 using /nix/store/iq2lnra6lg40lv2lryg4pa06ndk1saav-qemu-9.0.1/bin/qemu-system-x86_64 v9.0.1
 - Host:     Nobara Linux 40 (KDE Plasma) running Linux 6.8.12-201.fsync.fc40.x86_64 HOMEDTL
 - CPU:      12th Gen Intel(R) Core(TM) i7-12700K 
 - CPU VM:   host, 1 Socket(s), 4 Core(s), 2 Thread(s)
 - RAM VM:   8G RAM
 - BOOT:     EFI (macOS), OVMF (OVMF_CODE.fd), SecureBoot (off).
 - Disk:     macos-sonoma/disk.qcow2 (128G)
             Looks unused, booting from macos-sonoma/RecoveryImage.img
 - Recovery: macos-sonoma/RecoveryImage.img
 - Display:  SDL, VGA, GL (on), VirGL (off) @ (1280 x 800)
 - Sound:    intel-hda (hda-micro)
 - ssh:      On host:  ssh user@localhost -p 22220
 - 9P:       On guest: sudo mount_9p Public-adam
 - 9P:       On host:  chmod 777 /home/[user]/Public
             Required for macOS integration 👆
 - smbd:     On guest: smb://10.0.2.4/qemu
 - Network:  User (virtio-net)
 - Monitor:  On host:  nc -U "macos-sonoma/macos-sonoma-monitor.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:macos-sonoma/macos-sonoma-monitor.socket
 - Serial:   On host:  nc -U "macos-sonoma/macos-sonoma-serial.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:macos-sonoma/macos-sonoma-serial.socket
 - Process:  ERROR! Failed to start macos-sonoma.conf as macos-sonoma

ioctl(KVM_CREATE_VM) failed: 16 Device or resource busy
qemu-system-x86_64: failed to initialize kvm: Device or resource busy

The same happens with windows as well

quickemu --vm windows-11.conf
Error: couldn't find RGB GLX visual or fbconfig
~ ~
Quickemu 4.9.5 using /nix/store/iq2lnra6lg40lv2lryg4pa06ndk1saav-qemu-9.0.1/bin/qemu-system-x86_64 v9.0.1
 - Host:     Nobara Linux 40 (KDE Plasma) running Linux 6.8.12-201.fsync.fc40.x86_64 HOMEDTL
 - CPU:      12th Gen Intel(R) Core(TM) i7-12700K 
 - CPU VM:   host, 1 Socket(s), 4 Core(s), 2 Thread(s)
 - RAM VM:   8G RAM
 - EFI:      WARNING! /usr/share/OVMF/OVMF_CODE.fd is a symlink.
             Resolving to... /usr/share/edk2/ovmf/OVMF_CODE.fd
 - BOOT:     EFI (Windows), OVMF (/usr/share/edk2/ovmf/OVMF_CODE.fd), SecureBoot (off).
 - Disk:     windows-11/disk.qcow2 (64G)
             Just created, booting from windows-11/Win11_23H2_EnglishInternational_x64v2.iso
 - Boot ISO: windows-11/Win11_23H2_EnglishInternational_x64v2.iso
 - CD-ROM:   windows-11/virtio-win.iso
 - Display:  SDL, virtio-vga-gl, GL (on), VirGL (on) @ (1280 x 800)
 - Sound:    intel-hda (hda-micro)
 - ssh:      On host:  ssh user@localhost -p 22220
 - WebDAV:   On guest: dav://localhost:9843/
 - smbd:     On guest: smb://10.0.2.4/qemu
 - Network:  User (virtio-net)
 - Monitor:  On host:  nc -U "windows-11/windows-11-monitor.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:windows-11/windows-11-monitor.socket
 - Serial:   On host:  nc -U "windows-11/windows-11-serial.socket"
             or     :  socat -,echo=0,icanon=0 unix-connect:windows-11/windows-11-serial.socket
 - Process:  ERROR! Failed to start windows-11.conf as windows-11

ioctl(KVM_CREATE_VM) failed: 16 Device or resource busy
qemu-system-x86_64: failed to initialize kvm: Device or resource busy

Not sure how to proceed from here. Any advice or help will be greatly appreciated.

**Comment by FaridAliyev-dev** [comment]
Yeah so basically I ran to that error too. You just have to find the directery where the .conf file is and edit it. Watch this he shows how to do those stuff here: https://www.youtube.com/watch?v=Qa6y_CiyAMA&t=609s
